engine/src/test/scala/cromwell/RetryableCallsSpec.scala

```diff
@@ -27,25 +25,27 @@ class RetryableCallsSpec extends CromwellTestkitSpec with Mockito {
     }
   }
 
+  val customizedLocalBackendOptions =
+    """
+      |{
+      |  "backend": "customizedLocalBackend"
+      |}"""".stripMargin
```

Comment:
I actually had this exact same bug in Hermes for a while... it'd return valid parse trees even though it didn't consume the entire token stream.